File: ProyectoTicketeraLP2APi-main/ProyectoTicketeraLP2APi-main/demo/src/main/java/com/cibertec/demo/model/usuario.java
package com.cibertec.demo.model;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.sql.Date;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Entity
public class usuario {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int idUsuario;
    private String nombre;
    private String apellido;
    private String correo;
    private String contraseña;
    private String rol;

}

File: aplicacionTicketeraConsumidaDesdeAPi-main/aplicacionTicketeraConsumidaDesdeAPi-main/SERVICE_DESK/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SERVICE_DESK.Models
{
    public class Usuario
    {
        public int IdUsuario { get; set; }
        public string nombre { get; set; }
        public string apellido { get; set; }
        public string correo { get; set; }


        public string Rol { get; set; }
       
        public string contrasena { get; set; }







    }
}